hack/install_deps_rpm.sh

```diff
@@ -4,30 +4,28 @@
 # python3-pycurl is needed for Amazon SQS (boto lib), we need Fedora's rpm - installing it from pip results in NSS errors
 REQUIREMENTS_TXT='postgresql-devel python34-devel libxml2-devel libxslt-devel python34-requests python34-pycurl'
 # f8a_worker/process.py requirements
-REQUIRES='git /usr/bin/npm rubygems unzip tar file findutils koji rpmdevtools nodejs-packaging wget'
-# BinwalkTask DigesterTask
-# binwalk is installed from the latest release from https://github.com/devttys0/binwalk, since the package on pip is broken, and project documentation recommends `python setup.py install` as the recommended way.
-# REQUIRES_TASK='binwalk ssdeep'
+REQUIRES='git /usr/bin/npm unzip tar file findutils koji rpmdevtools nodejs-packaging wget'
```

Comment:
Does anybody remember why we have `koji` and `rpmdevtools` here? Maybe downstream task (?)